mlrun/db/httpdb.py

```diff
@@ -3956,11 +3958,21 @@ def store_alert_config(
         project = project or config.default_project
         endpoint_path = f"projects/{project}/alerts/{alert_name}"
         error_message = f"put alert {project}/alerts/{alert_name}"
-        if isinstance(alert_data, mlrun.common.schemas.AlertConfig):
-            alert_data = alert_data.dict()
+        if isinstance(alert_data, AlertConfig):
+            alert_data = alert_data.to_dict()
+        self._validate_alert(alert_data)
```

Comment:
Won't this be done anyway in the server side as part of the pydantic de-serialization? It's a bit faster to check it here rather than round-trip to the server, but since this is only useful if the user provided a dict then I'm not sure it's really needed.